src/Annotation/AbstractAnnotation.php

```diff
@@ -1,81 +0,0 @@
-<?php
-
-namespace Opstalent\CrudBundle\Annotation;
-
-use Doctrine\Common\Annotations\Annotation\Required;
-use Opstalent\CrudBundle\Exception\UnavailableActionException;
-
-/**
- * Class AbstractAnnotation
- * @author Szymon Kunowski <szymon.kunowski@gmail.com>
- * @package Opstalent\CrudBundle
- */
-abstract class AbstractAnnotation
```

Comment:
why are you removing this?

Reply:
Abstract Annotation (it's not really annotation btw) was splitted into two traits. I need this traits in different classes (From Model need only Available ActionTrait)
so Annotation and model should not extend same class but i need the same function so created traits.